index.jsx

```diff
@@ -335,7 +347,18 @@ exports.decorateHyper = (Hyper, { React }) => {
         }
 
         handleCwdClick(event) {
-            shell.openExternal('file://' + this.state.cwd);
+            if (globalConfig.hyperStatusLine.openInEditor) {
```

Comment:
How do you feel about getting rid of the `openInEditor` option and just testing if `editor` is provided?

Reply:
Sounds good.